papers/2024/GPAM/README.md

```diff
@@ -0,0 +1,73 @@
+
+[Full version](https://arxiv.org/abs/2306.07249)
+
+To make cryptographic processors more resilient against side-channel attacks,
+engineers have developed various countermeasures. However, the effectiveness of
+these countermeasures is often uncertain, as it depends on the complex interplay
+between software and hardware. Assessing a countermeasure’s effectiveness using
+profiling techniques or machine learning so far requires significant expertise
+and effort to be adapted to new targets which makes those assessments expensive.
+We argue that including cost-effective automated attacks will help chip design
+teams to quickly evaluate their countermeasures during the development phase,
+paving the way to more secure chips.
+
+In this paper, we lay the foundations toward such automated system by proposing
+GPAM, the first deep-learning system for power side-channel analysis that
+generalizes across multiple cryptographic algorithms, implementations, and
+side-channel counter-measures without the need for manual tuning or trace
+preprocessing. We demonstrate GPAM’s capability by successfully attacking four
+hardened hardware-accelerated elliptic-curve digital-signature implementations.
+We showcase GPAM’s ability to generalize across multiple algorithms by attacking
+a protected AES implementation and achieving comparable performance to
+state-of-the-art attacks, but without manual trace curation and within a limited
+budget. We release our data and models as an open-source contribution to allow
+the community to independently replicate our results and build on them.
+
+## List of datasets used in this paper
+
+### Newly captured datasets
+
+| Name    | Trace length | Examples in train | test  | holdout | Download size (in TB) | Download and usage instructions                                           |
+| ------- | ------------ | ----------------: | ----- | ------- | --------------------: | ------------------------------------------------------------------------- |
+| ECC CM0 | 1,6M         | 57,344            | 8,192 | 8,192   | 0.2                   | [ECC CM0](https://github.com/google/scaaml/tree/main/papers/datasets/ECC/GPAM) |
+| ECC CM1 | 5M           | 194,544           | 8,192 | 8,192   | 1.5                   | [ECC CM1](https://github.com/google/scaaml/tree/main/papers/datasets/ECC/GPAM) |
+| ECC CM2 | 10M          | 122,880           | 8,192 | 8,192   | 2.1                   | [ECC CM2](https://github.com/google/scaaml/tree/main/papers/datasets/ECC/GPAM) |
+| ECC CM3 | 17,5M        | 122,880           | 8,192 | 8,192   | 3.7                   | [ECC CM3](https://github.com/google/scaaml/tree/main/papers/datasets/ECC/GPAM) |
+
+### Publicly available datasets
+
+[REASSURE (H2020 731591) ECC Dataset](https://zenodo.org/records/3609789)
+Łukasz Chmielewski
+
+[ASCADv2](https://www.data.gouv.fr/en/datasets/ascadv2/)
+Loïc Masure and Rémi Strullu.
+Side-channel analysis against ANSSI’s protected AES implementation on ARM:
```

Comment:
I would add a line break here to make it easier to read.
1st line: dataset and authors
2nd line: description and link

WDYT?

Reply:
Not sure what description you have in mind, mostly that is a paper name. Like this?